backend/spacewalk-backend.spec

```diff
@@ -227,6 +227,8 @@ Requires:       %{name}-xmlrpc = %{version}-%{release}
 Requires:       systemd
 BuildRequires:  systemd
 %if 0%{?rhel}
+Requires:       python3-dnf
+Requires:       python3-hawkey
```

Comment:
Actually it's redundant as it seems `python3-hawkey` is already required by `python3-dnf`:
```
# rpm -q --requires python3-dnf
/usr/libexec/platform-python
dnf-data = 4.2.17-6.el8
libmodulemd >= 1.4.0
python(abi) = 3.6
python3-gpg
python3-hawkey >= 0.39.1
python3-libcomps >= 0.1.8
python3-libdnf
python3-libdnf >= 0.39.1
python3-rpm >= 4.14.2-35
rpmlib(CompressedFileNames) <= 3.0.4-1
rpmlib(FileDigests) <= 4.6.0-1
rpmlib(PartialHardlinkSets) <= 4.0.4-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
rpmlib(PayloadIsXz) <= 5.2-1
```

Reply:
I quickly talked with @vzhestkov: It seems that hawkeye is now a part of `libdnf` and obsolete. Thus I believe a direct requirement is not needed indeed.

See: https://github.com/rpm-software-management/hawkey

Reply:
@sbluhm no need to list `python3-hawkey` as required, explained in the comment above